### PR TITLE
EDU-427: Add Data Studios events to audit log table

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://seqera.atlassian.net/browse/EDU-427

Preview not working for this -- puzzling. But the _Files changed_ is correct.

### CODE REVIEW

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
This looks good to me! Preview also isn't working for me so once @robnewman approves we can merge and check in prod.

File: platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml [hunk 1]
```diff
@@ -15,6 +15,14 @@
   Resource:            'Data Explorer cloud buckets'
   Events logged:       'Add, edit, delete, hide, show'
   Note:                'Events for public and private buckets are logged.'
+-
+  Resource: 'Data Studios sessions'
+  Events logged: 'Add, started, connected, disconnected, stopped, deleted'
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml:20]:
```suggestion
  Events logged: 'Add, start, connect, disconnect, stop, delete'
```

Comment (robnewman) [reply platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml:20]: None of the other audit log entry verbs are past tense - they are all present tense.

File: platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml [hunk 2]
```diff
@@ -15,6 +15,14 @@
...
+  Note: 'Does not include temporary states (starting, stopping, deleting)'
+-
+  Resource: 'Data Studios custom environments'
+  Events logged: 'Build started, build succeeded, build failed'
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml:24]:
```suggestion
  Events logged: 'Build start, build success, build fail'
```

File: platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml [hunk 3]
```diff
@@ -15,6 +15,14 @@
   Resource:            'Data Explorer cloud buckets'
   Events logged:       'Add, edit, delete, hide, show'
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml:16]: Just noticed this - you can't `delete` a cloud bucket (!), but you can `remove` it from Data Explorer

```suggestion
  Events logged:       'Add, edit, remove, hide, show'
```

**Review by robnewman — APPROVED** [review]
For the sake of brevity/iterations I'm approving, but please incorporate my suggestions. Thanks!

**Pull request merged**